tests/tests_pytorch/strategies/test_colossalai_strategy.py

```diff
@@ -0,0 +1,291 @@
+# Copyright The PyTorch Lightning team.
```

Comment:
The filename should drop the `_strategy` suffix

Reply:
We have mix of both in our codebase. But sure